### ISSUE TITLE
Inaccurate screen reader alert appears in actions column of test queue after assigning a tester

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#797]
To reproduce:
1. Open test queue and go to any test plan.
2. In the testers column, open Assign testers menu and choose a tester.
3. After the assignment is made and the queue page is updated, read the actions column.

Result: After the buttons in the actions column, JAWS and NVDA see the message:

>You have been removed from this test run.

It comes from this element in the cell:

~~~
<div class="at-alert" role="status" aria-live="polite">You have been removed from this test run.</div>
~~~

This alert has two problems:
1. The message is inaccurate. It states a tester was removed when in fact a tester was assigned.
2. It is positioned in a distracting place in the DOM.

I recommend that screen reader alert messages be:
* Located someplace in the DOM where they are less likely to be distracting, e.g., end of the DOM.
* Short-lived, e.g., disappear after 10 to 15 seconds or at least when the user next interacts with any element on the page.

**Comment by mcking65** [comment]
Found another instance of the same problem in another place. As a tester, open a test plan from the test queue. Complete the version popup, after the first test loads, Press the next button to navigate to the next test in the plan.

The following irrelevant alert is displayed near the top of the page between the menu nav region and the test nav region:

>Your AT and Browser versions have been successfully saved.

That alert was relevant when the test runner first opened, but is not relevant when navigating to the next test. Like the one mentioned above, it should be:
* Placed at the end of the DOM.
* Removed shortly after created.

**Comment by mcking65** [comment]
Another more confusing screen reader alert is announced in the following scenario:

1. Open test queue and assign yourself to a plan.
2. Open the test runner.
3. Specify screen reader version and save and continue.
4. Activate Save and close in the runner.

5. Open the runner for the same test plan.

If using Chrome, the following alert is announced:

>Alert! The version of Chrome you have set is different from the one we have automatically detected, which is 116.0.0. This change doesn’t affect results that have already been submitted for this plan. However, results you submit during this session will be recorded with the versions specified in this form.

Both times I opened the test plan, the browser version was auto-detected, and the browser version was the same both times. The alert is confusing because it contains incorrect information.

It seems that the app is saving the major version number but detecting the full version number. The information presented in the browser version field in the dialog is confusing because .0.0 is appended to the major version. That gives users the impression that the full version is expected and saved. But that is not the case.

If the app is only saving the major version number, it should not display .0.0 in the version number field. This is probably one reason for the incorrect alert.

**Comment by ccanash** [comment]
The last part of the previous comment from @mcking65 won't be in scope for this task. I have created a separate task #810 to track that change

**Comment by mcking65** [comment]
The inaccurate alert described in the first comment this issue is now fixed, and it now appears at the end of the DOM. However, it is still very sticky. I assigned Paul to a test plan and the following text is at the end of the DOM:

>Paul-Clue now checked

That text node remains at the end of the DOM even after navigating to another page. Such messages are transient and the node should be removed to avoid confusion.

The other inaccurate and irrelevant alerts described in my 2nd and 3rd comments are still being announced.

This partial fix is helpful. The other remaining issues should not block deployment. However, I recommend keeping this issue open and in-progress.

**Comment by alflennik** [comment]
@ccanash since Matt pointed out a follow-up issue would you mind either creating a new issue (Alert persists as the end of the DOM) or reopening this one to track the second part of the task?

**Comment by mcking65** [comment]
Thank you, I verified that the information that was previously persistent is no longer persisting!
